bridges/snowbridge/primitives/core/src/rewards.rs

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+
+use frame_support::pallet_prelude::DispatchResult;
+pub type AccountIdOf<T> = <T as frame_system::Config>::AccountId;
+
+pub trait RewardLedger<T: frame_system::Config> {
+	// Deposit reward which can later be claimed by `account`
+	fn deposit(account: AccountIdOf<T>, value: u128) -> DispatchResult;
+}
```

Comment:
Suggest to refactor a bit as 
https://github.com/yrong/polkadot-sdk/blob/outbound-queue-v2/bridges/snowbridge/primitives/core/src/reward.rs

Reply:
Do you think we need to use Balance type?

Reply:
Made some changes over here: https://github.com/claravanstaden/polkadot-sdk/pull/1/commits/dd5ad4a259fa3fcd9efa798d11a5535e6595657f